### PR TITLE
feat: doc site

### PR DESCRIPTION
**Pull request by yetone**

### CODE REVIEW

File: docs/source/installation/migration.rst [hunk 1]
```diff
@@ -0,0 +1,127 @@
...
+
+    Because we removed all yatai component operators, yatai now does not automatically integrate logging and monitoring. See the `Observability </observability/index.html>`_ documentation for observability configuration.
+
+Migration steps
+---------------
+
+1. Backup PostgreSQL data
+"""""""""""""""""""""""""
+
+.. note:: If you use the external PostgreSQL, you need to skip this step.
```
Comment (ssheng) [inline docs/source/installation/migration.rst:38]:
```suggestion
.. note:: If you use an external PostgreSQL, you can to skip this step.
```

File: docs/source/installation/migration.rst [hunk 2]
```diff
@@ -0,0 +1,127 @@
...
+
+  pkill kubectl
+  kubectl port-forward --namespace yatai-system svc/yatai-postgresql 5433:5432 &
+  sleep 6
+  PGPASSWORD=$(kubectl -n yatai-system get secret yatai-postgresql -o jsonpath='{.data.postgresql-password}' | base64 -d) pg_dump -h localhost -p 5433 -U postgres -F t yatai > /tmp/yatai.tar
+
+2. Get MinIO environment variables
+""""""""""""""""""""""""""""""""""
+
+.. note:: If you use the external S3, you need to skip this step.
```
Comment (ssheng) [inline docs/source/installation/migration.rst:50]:
```suggestion
.. note:: If you use an external storage such as AWS S3, you can to skip this step.
```

File: docs/source/installation/migration.rst [hunk 3]
```diff
@@ -0,0 +1,127 @@
...
+.. note:: If you use the external PostgreSQL, you need to skip this step.
+
+.. code:: bash
+
+  pkill kubectl
+  kubectl port-forward --namespace yatai-system svc/yatai-postgresql 5433:5432 &
+  sleep 6
+  PGPASSWORD=$(kubectl -n yatai-system get secret yatai-postgresql -o jsonpath='{.data.postgresql-password}' | base64 -d) pg_dump -h localhost -p 5433 -U postgres -F t yatai > /tmp/yatai.tar
+
+2. Get MinIO environment variables
```
Comment (ssheng) [inline docs/source/installation/migration.rst:47]:
```suggestion
2. Get object store environment variables
```

File: docs/source/installation/migration.rst [hunk 4]
```diff
@@ -0,0 +1,127 @@
...
+.. code:: bash
+
+  export S3_ENDPOINT=minio.yatai-components.svc.cluster.local
+  export S3_ACCESS_KEY=$(kubectl -n yatai-components get secret yatai-minio-secret -o jsonpath='{.data.accesskey}' | base64 -d)
+  export S3_SECRET_KEY=$(kubectl -n yatai-components get secret yatai-minio-secret -o jsonpath='{.data.secretkey}' | base64 -d)
+  export S3_SECURE=false
+  export S3_BUCKET_NAME=yatai
+  export S3_REGION=i-dont-known
+
+3. Test MinIO connection
```
Comment (ssheng) [inline docs/source/installation/migration.rst:61]:
```suggestion
3. Test object store connection
```

File: docs/source/installation/migration.rst [hunk 5]
```diff
@@ -0,0 +1,127 @@
...
+
+  In v0.4.x, yatai component operators will automatically install dependencies of the component. But in v1.0.0, we removed this feature.
+
+  Some things will changes:
+
+  * No more integration of logging and monitoring
+
+    Because we removed all yatai component operators, yatai now does not automatically integrate logging and monitoring. See the `Observability </observability/index.html>`_ documentation for observability configuration.
+
+Migration steps
```
Comment (ssheng) [inline docs/source/installation/migration.rst:32]: Please clarify if migration requires Yatai system down time. @yetone

File: docs/source/installation/yatai_deployment.rst [hunk 6]
```diff
@@ -0,0 +1,472 @@
...
+
+Quick install
+------------------
+
+.. note:: This quick installation script can only be used for **development** and **testing** purposes
+
+This script will automatically install the following dependencies inside the :code:`yatai-deployment` namespace of the Kubernetes cluster:
+
+* cert-manager (if not installed)
+* metrics-server (if not installed)
```
Comment (ssheng) [inline docs/source/installation/yatai_deployment.rst:62]:
```suggestion
* cert-manager (if not already installed)
* metrics-server (if not already installed)
```

File: docs/source/installation/migration.rst (hunk 1 above)
Comment (yetone) [reply docs/source/installation/migration.rst:38]: @ssheng If using an external database, this must be skipped

File: docs/source/installation/migration.rst (hunk 2 above)
Comment (yetone) [reply docs/source/installation/migration.rst:50]: @ssheng  If using an external s3, this must be skipped

File: docs/source/yatai_wordlist.txt [hunk 7]
```diff
@@ -0,0 +1,164 @@
...
+txt
+unpickle
+url
+utf
+util
+vocab
+xgb
+xgboost
+yaml
+yatai
```
Comment (ssheng) [inline docs/source/yatai_wordlist.txt:163]: Should we ensure all occurrences of Yatai is capitalized?

**Review by ssheng — APPROVED** [review]

File: docs/source/yatai_wordlist.txt (hunk 7 above)
Comment (yetone) [reply docs/source/yatai_wordlist.txt:163]: @ssheng If yatai-deployment cannot be capitalized, it will be asymmetrical when put together with Yatai

**Comment by ssheng** [comment]
We can fix the details in future PRs. The docs as-is look great, especially the installation guides. Great work, @yetone!

**Pull request merged**